docs/tutorial/graphical.rst

```diff
@@ -208,6 +203,65 @@ Now that you have learned about the ManagedWindow, you have all of the basics to
       import pyqtgraph as pg
       pg.setConfigOption("useOpenGL", True)
 
+The filename and directory input
+################################
+
+By default, a ManagedWindow instance contains fields for the filename and the directory to control where the results of an experiment are saved.
+
+.. image:: pymeasure-fileinput.png
+    :alt: The filename and directory input widget
+    :width: 24%
+.. image:: pymeasure-fileinput_disabled.png
+    :alt: The filename and directory input widget, disabled to store the measurement to a temporary file
+    :width: 24%
+.. image:: pymeasure-fileinput_complete_filename.png
+    :alt: The filename and directory input widget showing the auto-complete for the filename
+    :width: 24%
+.. image:: pymeasure-fileinput_complete_directory.png
+    :alt: The filename and directory input widget showing the auto-complete for the directory
+    :width: 24%
+
+With the filename input, additionally a toggle (named :guilabel:`Write file` controlling whether the measurement is save at all, or should be stored in a temporary file, is displayed.
+When the toggle is ticked, the measurement is stored to a temporary file, and the other input fields are disabled.
+
+When the measurement is to be stored in to a certain file, ensure that the :guilabel:`Write file` is disabled.
```

Comment:
It doesn't seem right to me to check "write file" in order to **not** write to a file!

Reply:
Yes, you are right, this should reald "enabled". With your clarified sentence above, this one is unnecessary.